index.html

```diff
@@ -18,7 +21,7 @@
 </script>
 
 </head>
-<body>
+<body onload="loadProjects()">
```

Comment:
- [ ] On the mobile menu, There is an unnecessary gap above the header. Kindly try to fix this.

- [ ] On the mobile menu there is extra horizontal scrolling. Kindly check the below screenshot and fix it.

![image](https://user-images.githubusercontent.com/53019916/131711592-a0d19be8-b9ab-4193-a474-b5c21ab0af9e.png)

Reply:
Hi (@hemant-soni-vst-au4)

Kindly explain what you meant by unnecessary gap above the header. I am having difficulty understanding that part, if you can add an images as well. I would greatly appreciate it.

Reply:
- [ ] Hello @see-why. I have already added the images related to the issues part. I am adding new images, The area is marked with blue (as you can see below screenshot). This blank area should not be there as per Figma's design. This is happening in mobile view only and not on desktop view. If you still find it difficult to understand please let me know.

![image](https://user-images.githubusercontent.com/53019916/131738751-9dde0c05-bf70-44fd-8ed9-43b74dc012b3.png)

Reply:
Thank you,

I understand now.